### DISCUSSION TITLE
Using environment variables in options file

### DISCUSSION THREAD

**Discussion by MichalisDBA** [osalvador/ReplicaDB#66] [General]
Hello.

Can you give us an example of how to use environment variables in the options config file?

**Comment by osalvador** [comment]
Hi @MichalisDBA,

You can review the docs, there is an example: https://osalvador.github.io/ReplicaDB/docs/docs.html#31-using-options-files-to-pass-arguments

**Reply by MichalisDBA** [reply]
Hello again. 

What i am trying to achieve is how can i use variables inside a bash script and passing them inside the replicadb configuration file while calling replicadb with config file parameter inside a bash script. Is this possible?

**Reply by osalvador** [reply]
Hi @MichalisDBA ,

Yes, this is possible. All environment variables exported in bash script will be expanded in the ReplicaDB options-file. 

For example: `replicadb.conf`
```
######################## ReplicadB General Options ########################
mode=${MODE}
jobs=${JOBS}
fetch.size=${FETCH}
...
```

Your bash script: 

```bash
#!/usr/bin/env bash

export MODE=complete
export JOBS=2
export FETCH=100

replicadb --options-file replicadb.conf

```

**Reply by MichalisDBA** [reply]
Great thank you very much. Will try it!
